lib/perl/Genome/Model/Tools/Speedseq/Sv.pm

```diff
@@ -110,8 +109,13 @@ class Genome::Model::Tools::Speedseq::Sv {
                     doc => 'output LUMPY probability curves in VCF',
                     is_optional => 1,
                     tool_param_name => 'P',
-                }
-
+                },
+                verbose => {
+                    is => 'Boolean',
+                    doc => 'verbose',
```

Comment:
I agree.  I was merely using the description in the speedseq sv tool.  I updated the `doc` line a bit, but I'm not sure what else to include in the documentation.